tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -0,0 +1,321 @@
+                <Section title="Loaded Materials" textAlign="center">
+                  <LabeledList>
+                    {data.resource_data.map((resourceData: Resource) => (
+                      <LabeledList.Item
+                        key={resourceData.id}
+                        buttons={[
+                          <Button
+                            key="eject"
+                            icon="eject"
+                            onClick={() => act("material_eject", { "resource": resourceData.id })}
+                          />,
+                          <Button
+                            key="swap"
+                            icon="arrows-up-down"
+                            color={(swap !== resourceData.id) ? null : "green"}
+                            onClick={() => swapPriority(resourceData.id)}
+                          />,
+                        ]}
+                        label={toTitleCase(resourceData.name)}
+                        textAlign="center"
+                      >
+                        {resourceData.amount}
+                      </LabeledList.Item>
+                    ))}
+                  </LabeledList>
+                </Section>
+              </Stack.Item>
+              <Stack.Item>
+                <Section
+                  textAlign="center"
+                  title={"Fabricator Settings"}
+                >
+                  <LabeledList>
+                    <LabeledList.Item
+                      label={"Repeat"}
+                      buttons={<Button icon="repeat" onClick={() => toggleRepeat()}>Toggle Repeat</Button>}
+                      textAlign="center"
+                    >
+                      {data.repeat ? "On" : "Off"}
+                    </LabeledList.Item>
+                    <LabeledList.Item
+                      label="Speed"
+                    >
+                      <Slider
+                        minValue={1}
+                        value={data.speed}
+                        maxValue={data.hacked ? data.max_speed_hacked : data.max_speed_normal}
+                        step={1}
+                        stepPixelSize={100}
+                        disabled={data.mode !== "working"}
+                        onChange={(_e: any, value: number) => act("speed", { "value": value })}
+                      />
+                    </LabeledList.Item>
+                  </LabeledList>
+                </Section>
+                {data.manudrive.limit !== null ? (
+                  <Stack.Item>
+                    <Section
+                      title="Loaded Manudrive"
+                      buttons={
+                        <Button
+                          icon="eject"
+                          content="Eject"
+                          disabled={data.mode !== "ready"}
+                          onClick={() => act("manudrive", { "action": "eject" })}
+                        />
+                      }
+                      mb={1}
```

Comment:
This `Section` lives in a `Stack.Item`, is `mb` needed here? Should it be on the `Stack.Item` instead if it is needed?